xstate-chatbot/react-app/src/index.jsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Chat } from '@progress/kendo-react-conversational-ui';
+
+import { interpret } from 'xstate';
+import chatbotMachine from '../../nodejs/src/machine/seva';
+
+function MessageTemplate(props) {
+    let message = props.item.text;
+    // console.log(message);
+    if(typeof message == 'string')
+        message = message.replaceAll('\n', '<br/>');
+    else if(typeof message == 'object')
+        message = JSON.stringify(message);
+    let htmlToinsert = { __html: message };
+    return (
+        <div className="k-bubble">
+            <div dangerouslySetInnerHTML={htmlToinsert} />
+        </div>
+    );
+}
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.user = {
+            id: 1,
+            avatarUrl: "https://via.placeholder.com/24/008000/008000.png"
+        };
+        this.bot = { id: 0 };
+        this.state = {
+            messages: [
+            ]
+        };
+    }
+
+    componentDidMount() {
+        this.chatbotService = interpret(chatbotMachine.withContext ({  
+            chatInterface: this,
+            user: {"authToken":"xxxxx","refreshToken":"yyyyy","userInfo":{"id":24,"uuid":"xxxxx","userName":"9123123123","name":"John Doe","mobileNumber":"9123123123","emailId":"abcd@gmail.com","locale":"en_IN","type":"CITIZEN","roles":[{"name":"Citizen","code":"CITIZEN","tenantId":"pb"}],"active":true,"tenantId":"pb"},"userId":"xxxxx","mobileNumber":"9123123123","name":"John Doe","locale":"en_IN"},
+            extraInfo: {
+                tenantId: 'pb',
+                whatsAppBusinessNumber: "917834811114"

```

Comment:
Why is this not coming from config?

Reply:
The react-app is present only to make it easier to onboard new developers and facilitate the initial dialog development. It is only to be used on the developer's local machine(which has been clearly mentioned in README). So I haven't created any separate config files and the dummy values are present in the code.